source/features/move-description.tsx

```diff
@@ -0,0 +1,50 @@
+import {getOwnerAndRepo} from '../libs/utils';
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+
+async function init() {
+	const container = select('.repohead-details-container');
+	if (!container) {
+		return false;
+	}
+
+	const originalDesc = select("div.repository-content > div > div.f4 > span");
+	originalDesc.remove();
+
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const userInfo = await api.v4(
+		`{
+			repository(owner: "${ownerName}", name: "${repoName}") {
+				description
+				repositoryTopics(first: 100) {
```

Comment:
Did you mean to continue to pull all these other fields? Also, why do we need to query this instead of only moving it?

Reply:
Ah, that was leftover from trying to move the topics. We need the query because this allows the description to show up on any page that has the repo name.

Reply:
I'm really all for any feedback on this. The way tonsky designed this wasn't really well thought out (or explained if he did think about it), so I'm just trying to come up with something.